xercesc.sh

```diff
@@ -0,0 +1,33 @@
+package: XercesC
+version: v3.1.4
+source: https://github.com/PMunkes/XercesC
+tag: v3.1.4
+env:
+  XERCESC_INST_DIR: "$XERCESCINST"
```

Comment:
This is not really blocking, as most of the changes are already in shipdist. I am still working on a number of other recipes needed for FairShip, and this would only be blocking that.